pyqtgraph/palette.py

```diff
@@ -0,0 +1,72 @@
+from .Qt import QtGui
+
+
+def dark_QPalette():
+    RED = QtGui.QColor('#F00000')
```

Comment:
Oops, I only defined that for debugging :)
Could you delete it? 
Probably "BLACK", too, Although that has at least some usability.

Reply:
I was wondering what those were but I didn't want to make any changes. 

Done